plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -236,7 +236,10 @@ global proc mayaUsdTranslatorExport_updateDefaultPrimList() {
         return;
     }
 
-    string $currentDefaultPrim = `optionMenuGrp -q -value exportDefaultPrim`;
+    string $currentDefaultPrim = "";
+    if (`optionMenuGrp -q -numberOfItems exportDefaultPrim` > 0) {
+        $currentDefaultPrim = `optionMenuGrp -q -value exportDefaultPrim`;
+    }

```

Comment:
Also fix an error I noticed when opening the export options with only an empty stage. There is no default prim, so the optionMenu contains no entries.